website/docs/concepts/reading.mdx

```diff
@@ -275,6 +275,52 @@ If we used `ref.read` here, our button would still use the previous
 `StateController` instance, which was disposed and should no-longer be used.  
 Whereas using `ref.watch` correctly rebuilt the button to use the new `StateController`.
 
+### Using .update() to update previous state esily
```

Comment:
I don't like this being here, since:
- it's not about "reading"
- it's specific to `StateProvider`

Reply:
@rrousselGit  There's a typo in **esily** (easily) to be fixed before merging